README.md

```diff
@@ -1 +1,18 @@
-# proposals
\ No newline at end of file
+# Proposals
+This is an index of the proposed changes in _Bazel embedded_ ecosystem. 
+
+## Index
+
+### Under Review
+| Created | Title | Authors(s) | Category |
+|--------------|-------|------------|----------|
+
+### Draft
+| Created | Title | Authors(s) | Category |
+|--------------|-------|------------|----------|
+| 2021-11-24   | [Embedded Platforms](https://docs.google.com/document/d/1t_0NZhVUMnez4sbhcyxGBEOsxc2-7QSFeU1WX4HlaqM/edit?usp=sharing) | @silvergasp | Platforms |
+
+### Approved
+| Created | Title | Authors(s) | Category |
```

Comment:
Consider adding an "Approved" date column to track when the decision was finalized.

Reply:
Also maybe helpful to list an "Approvers" column through to make it clear who's approval we need for a proposal to be approved.